Week2/homework/Masoud/js-exercises/EX4-theReadingList.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+let books = [ 
+  { title : "ASTRONOMY FOR DUMMIES",
+    author : "STEPHEN P. MARAN",
+    alreadyRead : true
+  },
+
+  { title : "ASTRONOMY: A SELF-TEACHING GUIDE",
+    author : "DINAH L. MOCHE",
+    alreadyRead : false
+  },
+
+  { title : "THE UNIVERSE IN A NUTSHELL",
+    author : "STEPHEN HAWKING",
+    alreadyRead : true
+  }
+];
+
+for (let i =0 ; i < books.length ; i++) {
+  if(books[i].alreadyRead == true) {
+    console.log(books[i].title + ' by ' + books[i].author);
```

Comment:
As part of the requirement, this should be outside the loop. Move it outside to avoid duplicate code. This principle is called: DRY: Don't Repeat Yourself